tests/listsAndDropDowns.spec.ts

```diff
@@ -0,0 +1,84 @@
+import { test, expect } from "@playwright/test";
+
+test.beforeEach(async ({ page }) => {
+  await page.goto("/");
+  await page.getByText("Owners").click();
+  await page.getByText("Search").click();
+  await expect(page.locator("h2")).toHaveText("Owners");
+});
+
+test.describe("Lists and Drop Downs", async () => {
+  test("Validate selected pet types from the list", async ({ page }) => {
+    await page.getByText("George Franklin").click();
+
+    await expect(page.locator(".ownerFullName")).toHaveText("George Franklin");
+
+    await page.getByRole("button", { name: "Edit Pet" }).nth(0).click();
+
+    await expect(page.locator("h2")).toHaveText("Pet");
+
+    await expect(page.locator("#owner_name")).toHaveValue("George Franklin");
+
+    await expect(page.locator("#type1")).toHaveValue("cat");
+
+    const dropDownMenu = page.locator("#type");
+    await dropDownMenu.click();
+
+    const types = {
+      cat: "cat",
+      dog: "dog",
+      lizard: "lizard",
+      snake: "snake",
+      animal: "animal",
+      hamster: "hamster",
+    };
```

Comment:
Instead of looping through the object, it's better to loop through the array of values.
Also, instead of hardcoding the list of available pets in the code, you can first extract this list from the page into the array and then loop through it. 

Please refactor the code this way.

Reply:
done